sdk/resourcemanager/azure-resourcemanager-appservice/src/test/java/com/azure/resourcemanager/appservice/FunctionAppsTests.java

```diff
@@ -352,36 +351,81 @@ public void canCRUDLinuxFunctionAppPremiumDocker() {
     public void canCRUDLinuxFunctionAppJava11() throws Exception {
         rgName2 = null;
 
+        String runtimeVersion = "~4";
+
         // function app with consumption plan
         FunctionApp functionApp1 = appServiceManager.functionApps().define(webappName1)
             .withRegion(Region.US_EAST)
             .withNewResourceGroup(rgName1)
             .withNewLinuxConsumptionPlan()
             .withBuiltInImage(FunctionRuntimeStack.JAVA_11)
+            .withRuntimeVersion(runtimeVersion)
+            .withHttpsOnly(true)
+            .withAppSetting("WEBSITE_RUN_FROM_PACKAGE", FUNCTION_APP_PACKAGE_URL)
+            .create();
+        Assertions.assertNotNull(functionApp1);
+        assertLinuxJava(functionApp1, FunctionRuntimeStack.JAVA_11, runtimeVersion);
+
+        assertRunning(functionApp1);
+    }
+
+    @Test
+    public void canCRUDLinuxFunctionAppJava17() throws Exception {
+        rgName2 = null;
```

Comment:
why set rgName2 = null here?

Reply:
we only use rgName in this test.

just a hack so clean-up know it does not need to delete rgName2 resource group